doc/apiv3-documentation.apib

```diff
@@ -1474,6 +1478,94 @@ This is an example of how a form might look like. Note that this endpoint does n
                 "message": "The resource you are about to edit was changed in the meantime."
             }
 
+# Group MyPreferences
+
+## Linked Properties
+|  Link     | Description                                              | Type          | Constraints           | Supported operations |
+|:---------:|----------------------------------------------------------| ------------- | --------------------- | -------------------- |
+| self      | This MyPreferences                                       | MyPreferences | not null              | READ                 |
+| user      | The current user that this preference belongs to         | User          | not null              | READ                 |
```

Comment:
Two (minor) issues I am seeing:

1. This is the only existing link to the current user as of now... I'd probably add another one to the root resource as shortcut. Otherwise we won't add one any time soon and I don't feel that going over the preferences is the right way to determine the "self"
2. For the description I'd say "The ~~current~~ user that ...", because it is only by chance, that we are looking at our own preferences